src/components/BookLogic/BookForm.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+const BookForm = () => {
+  const [state, setState] = useState({
+    bname: '',
+    aname: '',
+  });
+
+  const handleChange = (e) => {
+    setState((state) => ({
+      ...state,
+      [e.target.name]: e.target.value,
+    }));
+  };
+
+  return (
+    <form>
```

Comment:
- [ ] Your `NewBookForm` component is required to have a submit button, However it does not require the functionality to be done yet 👍